NEW: Add shipments and reception stats on product stat tab

jyhere opened the pull request.

jyhere committed:
ADD: shipments and reception stats on product stat tab

eldy commented on htdocs/core/lib/product.lib.php:
```diff
@@ -571,6 +572,46 @@ function show_stats_for_company($product, $socid)
 	}
 
+	// Shipments
+	if (isModEnabled('expedition') && $user->hasRight('expedition', 'lire')) {
```
You shoul be able to use the english name now
 isModEnabled('shipment')

jyhere committed:
-FIX: product stats shipping / expedition code was not v20 friendly
-FIX: expeditiondet.elementtype default value should be "commande" (eg. Commande::$element)

jyhere committed:
Merge branch 'develop' into jyhere/product-statspage-add-shipments-receptions

jyhere committed:
FIX: phan errors

jyhere committed:
Merge branch 'jyhere/product-statspage-add-shipments-receptions' of github.com:jyhere/dolibarr into jyhere/product-statspage-add-shipments-receptions

jyhere committed:
FIX: phan errors

eldy committed:
Merge branch 'develop' into jyhere/product-statspage-add-shipments-receptions
